src/azure-cli/HISTORY.rst

```diff
@@ -7,6 +7,7 @@ unreleased
 ^^^^^^^^^^^^^^^^^^
 
 * Add 'az -v' as shortcut for 'az --version' (#2926)
+* Fixed underscore in cluster name (#3024)
```

Comment:
This should be in the HISTORY file for azure-cli-vm.

Alternatively, you can opt to not include this change in the HISTORY.rst since it is a change to help only.